pkg/tracker/apitracker.go

```diff
@@ -0,0 +1,280 @@
+/*
+Copyright © 2023 SUSE LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tracker
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"strings"
+
+	"github.com/Masterminds/log-go"
+	"github.com/containers/gvisor-tap-vsock/pkg/types"
+	"github.com/docker/go-connections/nat"
+)
+
+const (
+	GatewayBaseURL = "http://192.168.127.1:80"
+	HostSwitchIP   = "192.168.127.2"
```

Comment:
Please document why these can be constant (and I guess since they're exported, what they're for).